doc/src/conf.py

```diff
@@ -372,7 +372,13 @@
 \usepackage{letltxmacro}
 \LetLtxMacro\OldLaTeX\LaTeX
 \AtBeginDocument{\DeclareRobustCommand{\LaTeX}{\text{\OldLaTeX}}}
-'''
+\let\OldUnderscore\_
+\makeatletter
+\AtBeginDocument{\sbox\sphinxcontinuationbox{\spx@opt@verbatimcontinued}}
+\makeatother
+\protected\def\_{\OldUnderscore\discretionary{}{\sphinxafterbreak}{}}
```

Comment:
This mantra is to allow TeX to break after an underscore (it does work also if `\_` is encountered in math mode).